config/locales/ja.yml

```diff
@@ -417,3 +417,18 @@ ja:
   working_group:
     contact_details: 連絡先の詳細
     policies: 方針
+  worldwide_organisation:
```

Comment:
I've hopefully corrected the visual differences that need doing, but given the final design of this page is yet to be decided, I'm not sure how much more time its worth putting in making it perfect _now_. I'll follow this up with a commit for the translation content, as it will be a copy paste marathon, and I'd like to get opinions on the other changes while doing that.

Reply:
It's very hacky, but I've adapted my code for moving locale entries from one app to another for Government Frontend:

```bash
cd ~/govuk/whitehall
govuk-docker-run rake "translation:export:all[locale_export]"

cd locale_export
mkdir updated

for f in *.csv                               
do
  grep 'key,source,translation\|worldwide_organisation' $f > updated/$f              
done

cd ~/govuk/government-frontend
cp -r ~/govuk/whitehall/locale_export/updated ./

govuk-docker-run rake "translation:export:all[locale_export]"

cd locale_export
mkdir updated

for f in *.csv                               
do
  cat $f > updated/$f
  grep -v 'key,source,translation' ../updated/$f >> updated/$f
done

cd ..

govuk-docker-run rake "translation:import:all[locale_export/updated]"
govuk-docker-run rake "translation:normalize"
```

If you get any unexpected changes (e.g. unrelated keys being moved into alphabetical order), run `govuk-docker-run rake "translation:normalize"` in `government-frontend` before all this, then commit the changes as a separate commit which normalizes the locale files.